config.go

```diff
@@ -65,6 +85,7 @@ type Config struct {
 	// this provides natural fairness over time while controlling peak memory usage.
 	// Example: Setting to 20 limits memory to ~20 * 10k records * avg_record_size
 	maxConcurrentShards int
+	metricsConfig       MetricsConfig
```

Comment:
Looks like it's here _and_ it's in each stats receiver implementation. How is this field used here in top-level config?

Reply:
It's not, it's just passed down - I just felt that it should be a configuration of the app rather than buried in the stats receiver. Perhaps that's not the right pattern though

Reply:
I can't see how it's passed down, if we remove this field from here, everything still works, right?

I think if we have this field here, we should do enable/disable check layer above, sth like:

```go
if k.config.metricsConfig == nil || k.config.metricsConfig.EnableEventToClient {
	k.config.stats.EventToClient(*record.record.ApproximateArrivalTimestamp, record.retrievedAt)
} 
```

in `kinsumer.go`, not in statsd implemenation.

Reply:
Oooh I see what you mean, apologies! Yeah I agree. And actually, I dislike both 😂 

Prepping a follow-up PR to hopefully do something cleaner thanks for flagging!

Reply:
https://github.com/snowplow-devops/kinsumer/pull/35